helm/tweek/templates/infra.yaml

```diff
@@ -0,0 +1,195 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tweek-secrets-{{ .Release.Name }}
+data:
+  tweek_pfx_public_key: {{ .Values.secrets.tweek_pfx_public_key }}
+  tweek_ssh_public_key: {{ .Values.secrets.tweek_ssh_public_key }}
+  tweek_ssh_private_key: {{ .Values.secrets.tweek_ssh_private_key }}
+  minio_secret_key: {{ .Values.secrets.minio_secret_key }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tweek-redis-{{ .Release.Name }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tweek-redis-{{ .Release.Name }}
+    spec:
+      containers:
+      - name: redis
+        image: redis:alpine
+        args: ["--appendonly", "yes"]
+        ports:
+        - containerPort: 6379
+        volumeMounts:
+        - mountPath: /data
+          name: tweek-dbvolume-{{ .Release.Name }}
+      volumes:
+      - name: tweek-dbvolume-{{ .Release.Name }}
+        persistentVolumeClaim:
+          claimName: tweek-dbvolume-{{ .Release.Name }}
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: tweek-redis-{{ .Release.Name }}
+spec:
+  selector:
+    app: tweek-redis-{{ .Release.Name }}
+  ports:
+    - port: 6379
+      targetPort: 6379
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: tweek-dbvolume-{{ .Release.Name }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ .Values.dbvolume.size }}
+  storageClassName: {{ .Values.dbvolume.storageClass }}
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tweek-minio-{{ .Release.Name }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: tweek-minio-{{ .Release.Name }}
+    spec:
+      containers:
+      - name: minio
+        image: minio/minio
+        args:
+        - server
+        - /data
+        env:
+        - name: MINIO_ACCESS_KEY
+          value: AKIAIOSFODNN7EXAMPLE
```

Comment:
same in regards to minio credentials.

It might worth also adding support for external minio. (could be s3 bucket)